tests/Http/CampaignTest.php

```diff
@@ -308,6 +342,26 @@ public function testUpdatingACampaignWithInvalidStatus()
         $response->assertJsonValidationErrors(['contentful_campaign_id']);
     }
 
+    /**
+     * Test for updating a campaign with invalid status.
+     *
+     * PATCH /api/v3/campaigns/:campaign_id
+     * @return void
+     */
+    public function testUpdatingACampaignWithInvalidStatusWithGroupTypeId()
+    {
+        // Create a campaign to update.
+        $campaign = factory(Campaign::class)->create();
+
+        $response = $this->withAdminAccessToken()->patchJson('api/v3/campaigns/' . $campaign->id, [
+            'group_type_id' => "4", // This should be an integer
+        ]);
+
+        $response->assertStatus(422);
```

Comment:
It's very odd this one's failing with a 200 status - we should be receiving a 422 if the group_type_id doesn't exist. I wonder if changing the `group_type_id` to something alphanumeric like `cars` would cause the 422 ...